test/test_derivs.jl

```diff
@@ -8,38 +8,48 @@ using Transform
 import Synthetic
 import WCS
 
-import GSL.deriv_central
+import ForwardDiff
+
 
 println("Running derivative tests.")
 
+function do_deriv_test(f::Function, x::Float64, claimed_dx)
+    # TODO: really should call ForwardDiff.gradient just once, with a whole
+    # vector of x values, rather than once per parameter to test
+    fwd_deriv = ForwardDiff.gradient(f)([x,])[1]
+
+    info("got $fwd_deriv; expected $claimed_dx")
+
+    @test_approx_eq_eps significand(fwd_deriv) significand(claimed_dx) 1e-4
```

Comment:
That seems strange.  I'm testing the Hessian with autodiff and I'm using test_approx_eq.  Also, back when I had the autodiff in the objective wrapper I also recall that the results were pretty close.  I haven't looked closely at how you're doing it, but are you sure something isn't going wrong?